Add wrapper div around options & afterOptionsComponent

nickschot opened the pull request:
@cibernox this is the second feature request I made in Discord. This enables the following change: https://github.com/weddingshoppe/ember-model-select/pull/6/files#diff-6cd7a1538e5e1eb6201624d486136991 meaning the loading mask height is independent of any style changes that are made to the search component.

nickschot commented:
Heh, it seems something odd happened with rebasing. Just need to look at the power-select.hbs change (which is the div).

rwwagner90 commented:
@nickschot I realize this is super old, but could you please rebase and fix conflicts?

nickschot commented:
> @nickschot I realize this is super old, but could you please rebase and fix conflicts?

Not sure what was up with my branch, was a bit weird. But since it's a one liner I've just started afresh master again. Should be fixed.

rwwagner90 commented:
Thanks @nickschot! Is there any possibility this would break things for some people or not? I'm trying to help out maintaining things here, but I am not super familiar with the internals.

nickschot commented:
> seen this super old PR... @nickschot i have extracted your changes into a new PR and added into master.
> 
> I think this changes could be breaking for peoples which have written tests or added custom styles...
> 
> 
> 
> As we are close for releasing v9 version (and we have already breaking changes), i have marked this changes as breaking

Whoa! Rare for such old PRs. Nice one!
